tasks/cleanup-workspace/tests/pre-apply-task-hook.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
+TASK_PATH="$1"
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
`SCRIPT_DIR` can probably be deleted in most of these I'd think

Reply:
I checked and in every single file where it's defined, it's still used - even in this file where you added this comment. It's used for finding the location of mocks.sh:

```sh
yq -i '.spec.steps[0].script = load_str("'$SCRIPT_DIR'/mocks.sh") + .spec.steps[0].script' "$TASK_PATH"
```

I guess it could also be derived from the `$TASK_PATH`. I could do that if you'd prefer that. But I don't think it makes much of a difference - either way it will be a one liner.